src/streams/ReadableStream.ts

```diff
@@ -37,7 +37,7 @@ export interface Source<T> {
 	 * @returns A promise that resolves when the source's start operation has finished.  If the promise rejects,
 	 * 		the stream will be errored.
 	 */
-	start(controller: ReadableStreamController<T>): Promise<void>;
+	start?: (controller: ReadableStreamController<T>) => Promise<void>;
```

Comment:
fixed in https://github.com/msssk/core/commit/e01f17873d101eb384999251f17e6f0dcb478982